src/ds5/ds5-auto-calibration.cpp

```diff
@@ -502,7 +514,24 @@ namespace librealsense
                         cmd.data.push_back(p[0]);
                         cmd.data.push_back(p[1]);
                     }
-                    _hw_monitor->send(cmd);
+                    bool success = false;
+                    int iter =0;
+                    do
+                    {
+                        try
+                        {
+                            if (iter==0) // apply only in the first iteration
+                                std::this_thread::sleep_for(std::chrono::milliseconds(1000));   // TODO: sending shorter request
+                            auto res = _hw_monitor->send(cmd);                                  // is likely to fail with MIPI SKU
+                            success = true;
+                        }
+                        catch(...)
+                        {
+                            LOG_OCC_DEBUG("occ Save Statistics result failed");
+                            std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
Should this sleep remain?

Reply:
Correct, to provide time for FW to recuperate

Reply:
so, please add some comment there (we might remove it otherwise)